### PR TITLE
Add more handler event types

### PR DESCRIPTION
**Pull request by gscho**
Signed-off-by: gscho <greg.c.schofield@gmail.com>

## Description

Updates the event type list based on the code found in `lib/chef/event_dispatch/base.rb`

## Issues Resolved
N/A
## Check List

- [ ] Spell Check
- [ ] Local build
- [ ] Examine the local build
- [ ] All tests pass

### CODE REVIEW

**Review by tas50 — APPROVED** [review]
💯

File: themes/docs-new/layouts/shortcodes/dsl_handler_event_types.md [hunk 1]
```diff
@@ -191,10 +243,18 @@ method block by declaring it as the event type.
 <td>The Chef Infra Client is finished loading recipes.</td>
 </tr>
 <tr>
+<td><code>:cookbook_compilation_complete</code></td>
+<td>The Chef Infra Client completed all cookbook compliation phases.</td>
```
Comment (IanMadd) [inline themes/docs-new/layouts/shortcodes/dsl_handler_event_types.md]:
```suggestion
<td>The Chef Infra Client completed all cookbook compilation phases.</td>
```

Comment (gscho) [reply themes/docs-new/layouts/shortcodes/dsl_handler_event_types.md]: Thanks

**Review by kagarmoe — APPROVED** [review]
tenor-17177061

**Pull request merged**